shared/src/dispatchers/ses/sendBulkTemplatedEmail.js

```diff
@@ -44,12 +46,57 @@ exports.sendBulkTemplatedEmail = async ({
       Template: templateName,
     };
 
-    applicationContext.logger.info('Bulk Email Params', params);
+    await exports.sendWithRetry({ applicationContext, params });
+  } catch (err) {
+    applicationContext.logger.error(`Error sending email: ${err}`, err);
+  }
+};
+
+/**
+ * Sends the email via SES with a MAX_SES_RETRIES = 10;
```

Comment:
Currently set to `7` above. Added a comment below about maybe sticking this on the constants.